module-gui/gui/core/FontManager.cpp

```diff
@@ -96,28 +99,60 @@ namespace gui
         }
     }
 
-    std::vector<std::string> FontManager::getFontsList()
+    std::map<std::string, std::string> FontManager::getFontsList()
```

Comment:
as we change Font Manager - can we add some UT here? i.e. if all loaded are loaded, what happens when json states that there is font but there is no such font on disc etc
i.e. here: module-gui/test/test-catch-text/test-gui-Font.cpp